tasks/main.yml

```diff
@@ -25,8 +31,24 @@
     value: 'true'
     vtype: select
 
+- name: Copy the Oracle Java archive from our S3 bucket
+  aws_s3:
+    bucket: oracle-java-archives
```

Comment:
Since bucket names must be unique across AWS, shouldn't we make this bucket name a variable, to make things a bit easier for future users of this role (outside of our organization)?

Reply:
Yes.  I will do that now.

Reply:
Remedied in 935a1b9.

Reply:
Looks great!  🔭